### PR TITLE
Small refactors

### PR DESCRIPTION
**Pull request by vcarl**
* Allow arrays of channel IDs/handlers in `addHandler`
* Rename `duration` -> `seconds` in `sleep()`, for clearer intelligence
* Use `CHANNELS.jobBoard` instead of string id

(This is a stacked diff off #145)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -77,11 +78,27 @@ export type ChannelHandlersById = {
...
-const addHandler = (channelId: string, channelHandlers: ChannelHandlers) => {
-  channelHandlersById[channelId] = [
-    ...(channelHandlersById[channelId] || []),
-    channelHandlers,
-  ];
+const addHandler = (
+  oneOrMoreChannels: string | string[],
+  oneOrMoreHandlers: ChannelHandlers | ChannelHandlers[],
+) => {
+  const channels =
```
Comment (dylangarcia) [inline src/index.ts:85]: `const channels = [].concat(oneOrMoreChannels);` is an easy way to do this btw

File: src/index.ts [hunk 2]
```diff
@@ -77,11 +78,27 @@ export type ChannelHandlersById = {
...
+      : oneOrMoreChannels;
+  const handlers =
+    oneOrMoreHandlers instanceof Array
+      ? oneOrMoreHandlers
+      : [oneOrMoreHandlers];
+
+  channels.forEach((channelId) => {
+    const existingHandlers = channelHandlersById[channelId];
+    if (existingHandlers) {
+      existingHandlers.concat(handlers);
```
Comment (dylangarcia) [inline src/index.ts:97]: This is a floating concat - `push` instead?

Comment (vcarl) [reply src/index.ts:97]: whoop that's what it started as, good catch

File: src/index.ts (hunk 1 above)
Comment (vcarl) [reply src/index.ts:85]: TS complains, at least

Comment (dylangarcia) [reply src/index.ts:85]: TS is stupid

**Review by nickmccurdy — APPROVED** [review]

**Pull request merged**